### DISCUSSION TITLE
repo.get_codescan_alerts()

### DISCUSSION THREAD

**Discussion by mariolasagna007** [PyGithub/PyGithub#2357] [General]
Hi there. I'm trying to pull a list of code scan alerts using this example:

```
>>> repo = g.get_repo("PyGithub/PyGithub")
>>> codescan_alerts = repo.get_codescan_alerts()
>>> for alert in codescan_alerts:
...     print(alert.number, alert.created_at, alert.dismissed_at)
...     print("  ", alert.tool.name, alert.tool.version, alert.tool.guid)
...     print("  ", alert.rule.name alert.rule.security_severity_level alert.rule.severity)
...     print("    ", alert.rule.description)
...     print("  ", alert.most_recent_instance.ref, alert.most_recent_instance.state)
...     print("    ", alert.most_recent_instance.location)
...     print("    ", alert.most_recent_instance.message['text'])
```

Unfortunately, I'm getting the following error:

>>> codescan_alerts = repo.get_codescan_alerts()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Repository' object has no attribute 'get_codescan_alerts'

I checked the class with dir() and the function `get_codescan_alerts` is not really there. I'm using the g = Github(access_token) for authentication.

Any thoughts?

**Comment by nirliberman-devocean** [comment]
Can you please create a new release that contains get_codescan_alerts?
